### ISSUE TITLE
[FEATURE] Chat bot

### ISSUE THREAD

**Issue by cnu1812** [cnu1812/Community-Library#32]
### Description

It would be great if we have a chatbot, the visitors don't feel static when they visit the site! Something like this 
![61af90abf3afa69e84062732_chat-example (2)](https://user-images.githubusercontent.com/75531528/193070031-b0fe2956-422b-4541-9704-f7261e3c7e08.png)

**Comment by prajwal-144** [comment]
Hey! I would like to contribute to this issue.

**Comment by cnu1812** [comment]
> Hey! I would like to contribute to this issue.

Hi, @prajwal-144 , glad to assign this issue to you. Need any help, feel free to ping me.

**Comment by cnu1812** [comment]
Hi @prajwal-144 just wanted to ask, howz work is going on? If you stuck which bot to use, these are my examples typebot and chatwoot bot. If you need any help plz feel free to ping me ☺️

**Comment by prajwal-144** [comment]
I was working on building a chatbot from scratch using pytorch and tensorflow.

**Comment by cnu1812** [comment]
> I was working on building a chatbot from scratch using pytorch and tensorflow.

Okay, this is gonna be super awesome. All the best! Please feel free to reach me if you need any help.

**Comment by prajwal-144** [comment]
Hey @cnu1812 I am almost done with it just a few optimizations left.
I am not that familiar with frontend or backend. I have deployed few models with Flask but not familiar with JavaScript.
So how should I go about this?

**Comment by prajwal-144** [comment]
Hey @cnu1812 I have finished building the chatbot. I visited the Community-Library website and inferred from the same to build the bot. It is up and running good in the command prompt. I trained it on 1000 epochs and the Loss is around 0.006.
Should I deploy it using Flask or should I create PR with the work done now and then maybe you can have a look and suggest the required changes.
Thank you.

**Comment by cnu1812** [comment]
> Hey @cnu1812 I have finished building the chatbot. I visited the Community-Library website and inferred from the same to build the bot. It is up and running good in the command prompt. I trained it on 1000 epochs and the Loss is around 0.006. Should I deploy it using Flask or should I create PR with the work done now and then maybe you can have a look and suggest the required changes. Thank you.

Hi @prajwal-144 nice to hear this from you. First deploy it using flask, then test the chatbot at localhost. If it's up and running without any errors, plz raise a pr. And sorry for late reply 😃
